### PR TITLE
Docker yarn fix - now installs natively VS NPM 

### PR DESCRIPTION
**Pull request by trexdinosaur**
I discussed this with @uttu357 , about changing the Dockerfile to now install yarn via the native method, rather than NPM.

Thanks.

### CODE REVIEW

File: docker/staging/nodejs/Dockerfile [hunk 1]
```diff
@@ -11,9 +11,10 @@ COPY ./docker/staging/nodejs/container-start.sh /code/
 
 # Install Prerequisites
 RUN npm cache clean -f
-
-RUN npm install -g yarn
-RUN yarn install
+RUN apt-get install apt-transport-https
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update && apt-get install yarnRUN yarn install
```
Comment (uttu357) [inline docker/staging/nodejs/Dockerfile]: Fix this line.

File: README.md [hunk 2]
```diff
@@ -40,7 +40,9 @@
 7. Open a new terminal window with node(>=6) and ruby(gem) install on your machine and type
     ```
     cd frontend
-    sudo npm install -g yarn
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/ya
```
Comment (uttu357) [inline README.md]: Fix this line.

**Comment by trexdinosaur** [comment]
Hi, I have fixed it. However, I was considering that maybe it might be better just to put yarn as a requirement, rather than the instructions, because this is not a cross platform way to install it, it only works on linux. 

Thanks.

**Comment by uttu357** [comment]
Yeah. In the Readme remove the instruction and add link to Yarn installation instructions. We can keep the new way of installing Yarn in Dockerfiles

**Comment by gautamjajoo** [comment]
@trexdinosaur Any update on this?

**Comment by pavanagrawal123** [comment]
All changes are reflected, @uttu357 needed to setup travis

**Comment by uttu357** [comment]
Closed due to inactivity. The change in README.md is still left.

**Pull request closed without merging**